property.go

```diff
@@ -174,7 +234,7 @@ func ParseProperty(contentLine ContentLine) (*BaseProperty, error) {
 			t := r.IANAToken
 			r, np, err = parsePropertyParam(r, string(contentLine), p+1)
 			if err != nil {
-				return nil, fmt.Errorf("parsing property %s: %w", t, err)
+				return nil, fmt.Errorf("%s %s: %w", ParsingPropertyError, t, err)
```

Comment:
We won't be able to catch `ParsingPropertyError` `error.Is` and `error.As` now support multiple errors, so we can use `%w` twice, past a certain go version... Since 1.20: https://www.reddit.com/r/golang/comments/z870te/multiple_error_wrapping_is_coming_in_go_120/

Reply:
I guess this technical is a bug